catalyst_voices/packages/internal/catalyst_voices_blocs/lib/src/proposal/proposal_cubit.dart

```diff
@@ -46,45 +44,45 @@ final class ProposalCubit extends Cubit<ProposalState>
 
       final proposal = await _proposalService.getProposal(ref: ref);
       final comments = _buildComments();
-
-      if (isClosed) {
-        return;
-      }
-
-      final commentsSort = state.data.commentsSort;
-
-      final proposalViewData = _buildProposalViewData(
-        hasActiveAccount: _userService.user.activeAccount != null,
-        proposal: proposal,
-        comments: comments,
-        commentSchema: _buildCommentTemplate().document.schema,
-        commentsSort: commentsSort,
-        isFavorite: false,
-      );
+      final isFavorite = await _proposalService
+          .watchIsFavoritesProposal(
+            ref: ref,
+          )
+          .first;
```

Comment:
Somehow this feels too verbose. Like taking too much vertical space for the "small thing" it does

```suggestion
      final isFavorite =
          await _proposalService.watchIsFavoritesProposal(ref: ref).first;
```

Reply:
well i don't like but i'll do the change because later we may change line width so it fits :D